templates/1000-missing.html

```diff
@@ -4,8 +4,9 @@
 {% markdown %}
 # Missing theorems from [Freek Wiedijk's](https://www.cs.ru.nl/~freek/) [1000+ theorems project](https://1000-plus.github.io/)
 These theorems are not yet formalized in Lean (or, these formalisations are not entered in the database yet).
+Currently there are {{hundred_theorems|rejectattr('formalized')|list|length}} of them.
```

Comment:
I like this addition, but careful: "hundred_theorems" should be updated, right?

Reply:
```suggestion
Currently there are {{thousand_theorems|rejectattr('formalized')|list|length}} of them.
```